### ISSUE TITLE
become_flags are sudo-specific

### ISSUE THREAD

**Issue by fabianfreyer** [debops-contrib/ansible-checkmk_server#65]
When `become_method` is not `sudo`, the following `become_flags` don't really make sense:

https://github.com/debops-contrib/ansible-checkmk_server/blob/22149b9d5ae6929af6af752f7155341521409b65/tasks/site.yml#L146

https://github.com/debops-contrib/ansible-checkmk_server/blob/22149b9d5ae6929af6af752f7155341521409b65/tasks/site.yml#L179

e.g. for `su`, the corresponding flag should be `-l`.
If there is a hard dependency on `sudo`, then IMHO `become_method` should be specified for those tasks.

**Comment by ganto** [comment]
Good point, thanks. This role urgently needs a general overhaul, I'll definitely add this to the list of stuff being looked at. If it's urgent for you to be fixed, I'll gladly accept a PR. 😃
